packages/cli/openapi-ir-to-fern/src/__test__/__snapshots__/flagright.test.ts.snap

```diff
@@ -5606,6 +5606,7 @@ In order to make individual events retrievable, you also need to pass in a uniqu
     "auth": "x-api-key",
     "auth-schemes": {
       "x-api-key": {
+        "env": undefined,
```

Comment:
@armandobelardo same thing here for `env` on header schemes -- shouldn't see it come through as undefined

Reply:
It mirrors the other props in headers
<img width="543" alt="image" src="https://github.com/fern-api/fern/assets/11140328/fa24bab7-948a-43f3-b0f9-5e057e454ea6">

Do you want both changed?

Reply:
yeah thats a bug

Reply:
Updated both!